mosdepth.nim

```diff
@@ -872,6 +884,7 @@ Other options:
   -F --flag <FLAG>                  exclude reads with any of the bits in FLAG set [default: 1796]
   -i --include-flag <FLAG>          only include reads with any of the bits in FLAG set. default is unset. [default: 0]
   -x --fast-mode                    dont look at internal cigar operations or correct mate overlaps (recommended for most use-cases).
+  -a --fragment-mode                count the coverage of the full fragment based on the insert size (proper pairs only).
```

Comment:
replace "based on the insert size" with "including the insert"

Reply:
"including the _full_ insert" perhaps to signal the difference?

Reply:
Used "full" but let me know if I should remove it